src/components/NcSelect/NcSelect.vue

```diff
@@ -724,12 +704,18 @@ export default {
 
 		/**
 		 * Input element id
-		 *
-		 * @see https://vue-select.org/api/props.html#inputid
 		 */
 		inputId: {
 			type: String,
-			default: null,
+			default: () => `select-input-${GenRandomId()}`,
+		},
+
+		/**
+		 * Visible label for the input element
+		 */
+		inputLabel: {
+			type: String,
+			default: t('Options'),
 		},
```

Comment:
With a default value, it is a breaking change. It changes design and size of existing selects.


Before | After
---|---
![image](https://github.com/nextcloud-libraries/nextcloud-vue/assets/25978914/eca6279f-6d38-4593-97c3-0b1c5e328110) | ![image](https://github.com/nextcloud-libraries/nextcloud-vue/assets/25978914/1ecfd529-4d8c-42d5-a4a3-e2d0822ec773)